### ISSUE TITLE
Pin toots of other accounts

### ISSUE THREAD

**Issue by PaulFree14** [tootsuite/mastodon#5182]
...hmm why can't I pin toots from other accounts? I mean I like to pin toots, but  maybe someone else said something I'd like to highlight. 

Or I want to highlight a toot from an secound account I'm using.

**Comment by ng-0** [comment]
Can you be more specific?

Right now this reads like you want bookmarks.

**Comment by PaulFree14** [comment]
I mean I can pin my own toots. 
But I can't pin toots to my profile which have been posted by another account then mine. 

And I thought if would be great if I also could pin toots from others. 

(bookmarks would be also lovly :-). but that's not what I meant here)

**Comment by Gargron** [comment]
You can't pin toots from other accounts and you won't be able to because it opens up various forms of abuse. Permanently displaying someone else's status on your profile is something that should require consent, and I don't believe it's worthwhile to build out the functionality for that. Sorry.

**Comment by PaulFree14** [comment]
I agree that it should require consent, but from my point of view it's given once the person tooting chooses to make the post public available. 
If ppl don't want to have posts public they can simply choose 'private' or 'direct'. 

consent is there (imho), just what needs is 'allow all toots to be pinned except private, or direct, maybe also the 'unlisted'

**Comment by Cassolotl** [comment]
>I agree that it should require consent, but from my point of view it's given once the person tooting chooses to make the post public available.

Noooooooooooooooo

**Comment by PaulFree14** [comment]
that just confuses me. 
If I make something public available, thought it should be available for the public. 
I agree that this could open the door for abuses of it, but just as any pined toot does. 
I could just @ anybody into it with some bad comments. 
The only reason I don't is because I don't like it. 

but yeah it's also not on my 'must have list'. :)

**Comment by ng-0** [comment]
A closing remark: Consent is never the absence of an explicit statement or the assumption that you agree on something.
Consent must be given explicitly, for each occasion, can be revoked any time and can take many individual forms.
This reminds me of the pamphlet on consent I wanted to scan and upload
